src/main/java/com/sports/server/command/leagueteam/presentation/LeagueTeamController.java

```diff
@@ -0,0 +1,27 @@
+package com.sports.server.command.leagueteam.presentation;
+
+import com.sports.server.command.leagueteam.application.LeagueTeamService;
+import com.sports.server.command.leagueteam.dto.LeagueTeamRegisterRequest;
+import com.sports.server.command.member.domain.Member;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/manager/leagues/{leagueId}/teams")
```

Comment:
``/manager`` 만 남겨 놓고 나머지는 아래 ``@PostMapping``에 내리면 어떨까?

```suggestion
@RequestMapping("/manager")
```